gc/base/standard/Scavenger.cpp

```diff
@@ -355,6 +355,13 @@ MM_Scavenger::masterSetupForGC(MM_EnvironmentStandard *env)
 
 	_doneIndex = 0;
 
+	if ((NULL != _extensions->_tenureTLHRemainderTop) && (NULL != _extensions->_tenureTLHRemainderBase)){
```

Comment:
put a comment like 'restore master thread TLH remainder boundaries, preserved at the end of previous scavenge cycle'